packages/apollos-church-api/config.yml

```diff
@@ -131,6 +137,10 @@ ROCK_MAPPINGS:
 
   SERMON_CHANNEL_ID: 19
 
+  SERVING_GROUP_TYPE_ID: 23
```

Comment:
I don't think this one is right, the other two are though.

@briankalwat do you know what our group type ID for serving groups is?

Reply:
@redreceipt there are many serving group types. KS vol, Fuse vol, etc. Do you want all of them?